drivers/gpu/drm/panel/panel-ilitek-ili9881d.c

```diff
@@ -0,0 +1,309 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * ILI9881D panel driver
+ * 
+ * Copyright (c) 2020 Seeed Studio
+ * Zhangqun Ming<north_sea@qq.com>
+ */
+#include "panel-reTerminal.h"
+
+
+static struct mipi_dsi_device *ili9881d_dsi = NULL;
+
+static const struct drm_display_mode ili9881d_modes = {
+	.clock		= 62712 /*73164*/ /*83616*/ /*94068*/ /*104520*/,
+
+	.hdisplay	= 720,
+	.hsync_start= 720 + 10,
+	.hsync_end	= 720 + 10 + 20,
+	.htotal		= 720 + 10 + 20 + 30,
+
+	.vdisplay	= 1280,
+	.vsync_start= 1280 + 10,
+	.vsync_end	= 1280 + 10 + 20,
+	.vtotal		= 1280 + 10 + 20 + 30,
```

Comment:
This timing looks identical to lhr050h41_default_mode in drivers/gpu/drm/panel/panel-ilitek-9881c.c
Are they actually any different?